src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -4,6 +4,7 @@
 
 Rule ID | Category | Severity | Notes
 --------|----------|----------|-------
+CA2261 | Usage | Warning | DoNotUseConfigureAwaitWithSuppressThrowing, [Documentation](https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2250)
```

Comment:
Should this be https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2261 ?

Reply:
Yes. No idea how it ended up like that.